jupyter-on-gke/README.md

```diff
@@ -36,13 +36,20 @@ Preinstall the following on your computer:
 
 3. Once the image is built, navigate to `ai-on-gke/jupyter-on-gke/`
 
-4. Edit `variables.tf` with your GCP settings. The `<your user name>` that you specify will become a K8s namespace for your Jupyterhub services.
+4. Edit `variables.tf` with your GCP settings. The `<your user name>` that you specify will become a K8s namespace for your Jupyterhub services. For more information about what the variabls do visit [here](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/jupyter-on-gke/variable_definitions.md)
 **Important Note:**
 If using this with the Ray module (`ai-on-gke/ray-on-gke/`), it is recommended to use the same k8s namespace
 for both i.e. set this to the same namespace as `ai-on-gke/ray-on-gke/user/variables.tf`.
 If not, set `enable_create_namespace` to `true` so a new k8s namespace is created for the Jupyter resources.
-**Note:**
-We allow user to set their own domains, in the `variables.tf` file. Since we are also using an Ingress Object, it is required for the Ingress to also have specifiy the name of the global static address.
+
+5. If you have not enabled IAP or created a Brand for your project, you can proceed to the next step. If not, ensure that the following variables within `variables.tf` are set:
+
+    * enable_iap_service - Enables the IAP service API
```

Comment:
These are enabled by default, right? So why do we need to call this out?
Should this say "if you want to bring your own brand and/or IAP client, then make sure you set the below vars"?

Reply:
So with the IAP and Brand creation, if it was done before ie a brand is already created or the IAP api is already enabled,  terraform returns an error if you try to re-create/re-enable it. But ill make it clearer.